cmd/nerdctl/run.go

```diff
@@ -145,7 +145,10 @@ func setCreateFlags(cmd *cobra.Command) {
 		return []string{"host", "private"}, cobra.ShellCompDirectiveNoFileComp
 	})
 	cmd.Flags().String("cpuset-cpus", "", "CPUs in which to allow execution (0-3, 0,1)")
+	cmd.Flags().String("cpuset-mems", "", "MEMs in which to allow execution")
```

Comment:
Plz add an example in flag détails like we did for `cpuset-cpus`

Reply:
thank you very much, is it necessary to add example? here is docker command
```
docker run --help|grep cpu
      --cpu-period int                 Limit CPU CFS (Completely Fair Scheduler) period
      --cpu-quota int                  Limit CPU CFS (Completely Fair Scheduler) quota

```

Reply:
yes, that may help nerdctl users  ;)